plugin/braceless.vim

```diff
@@ -54,16 +54,12 @@ function! s:enable(...)
 
   if !empty(g:braceless#key#jump_prev)
     execute 'map <buffer> ['.g:braceless#key#jump_prev.' <Plug>(braceless-jump-prev-n)'
-    execute 'map <buffer> g'.g:braceless#key#jump_prev.' <Plug>(braceless-jump-prev-n-indent)'
```

Comment:
I thought about this quite a bit and I honestly think it should just be dropped.
If anyone misses it they can always remap it and `g]` was never documented as the primary key binding.

That being said I'll do as you wish.